pkg/util/certificates.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2023 The KEDA Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"crypto/x509"
+	"fmt"
+	"os"
+	"path"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+const customCAPath = "/custom-cas"
```

Comment:
Nit: I don't like the dash in the name, what about `/custom/certs/ca` or /custom/ca` or `/custom/cas`  or something like that?

It is fine, if you prefer to keep the original name, maybe it is only me 😄

Reply:
it's just a mounting path, the name isn't important

Reply:
Updated to `/custom/ca`